external/fv3fit/fv3fit/keras/_models/models.py

```diff
@@ -91,6 +94,9 @@ class DenseHyperparameters(Hyperparameters):
     loss: str = "mse"
     save_model_checkpoints: bool = False
     nonnegative_outputs: bool = False
+    packer_config: PackerConfig = dataclasses.field(
```

Comment:
Not an issue with/for your PR, but it continues to feel weird to me that we have a configuration container with one element which is another configuration container.